local-lib/testlib/testlib/typical.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+import configparser
+import feedparser
+import time
+import util
+import variables as var
+import json
+
+#TODO:
+# commands(**kargs) since __init__ self.bot
+# send private messages from tasks
+# install dependencies like feedparser along with this module
+
+route_map = {}
+def command(trig, methods=['handle']):
+    def inner_decorator(f):
+        if trig not in route_map:
+            route_map[trig] = {}
+            for method in methods:
+                route_map[trig][method] = f
+        return f
+    return inner_decorator
+
+
+class TestLib():
```

Comment:
I think this Lib does something like reading RSS feed. Can you elaborate more? And if this Lib does do some thing, then we should give it a meaningful name like RSSLib.

Reply:
This lib was pure for example/testing, I could bring it down as suggested being the example RSSLib